awx/plugins/inventory/tower.py

```diff
@@ -54,6 +54,13 @@ def parse_configuration():
     host_name = os.environ.get("TOWER_HOST", None)
     username = os.environ.get("TOWER_USERNAME", None)
     password = os.environ.get("TOWER_PASSWORD", None)
+    ignore_ssl = False
+    ssl_negative_var = os.environ.get("TOWER_IGNORE_SSL", None)
+    if ssl_negative_var:
+        ignore_ssl = ssl_negative_var.lower() in ("1", "yes", "true")
+    else:
+        ssl_positive_var = os.environ.get("TOWER_VERIFY_SSL", None)
+        ignore_ssl = ssl_positive_var.lower() not in ('true', '1', 't', 'y', 'yes')
```

Comment:
I'm really unclear on this code change; isn't all of the work of setting `ignore_ssl` just being undone on line 64 below?

Reply:
`ignore_ssl = os.environ.get("TOWER_IGNORE_SSL", "1").lower() in ("1", "yes", "y", "true", "t")

Reply:
oh, right, that was a bug. Good catch.

This isn't the same environment variable. TOWER_IGNORE_SSL vs. TOWER_VERIFY_SSL, one of them comes from tower-cli, and one of them needs continued support for users who have already hacked their systems to use the tower inventory source.

Reply:
...and the reason we want TOWER_VERIFY_SSL to work is so that there's shred of hope that users can practically use the tower modules with the built-in tower credential type. I don't know about security, but it probably helps that these requests will probably be targeting localhost in the first place.

Reply:
Okay, this makes more sense to me.